README.md

```diff
@@ -105,6 +105,10 @@ git checkout openshift-ansible-3.0.47-6
 ```
 
 ## Usage
+
+### Pick your AWS Region
+You have to pick which region to deploy your OpenShift installation.   Specify or modify 'run.py' with the correct AWS region and corresponding AMI. 
+
```

Comment:
You don't but then you need to push instructions that  you need to create the ec2 key in a SPECIFIC region for it to work.   I was trying to allow others to pick their region since not all of us are physically located in the default east region.